app/views/hyrax/base/_citations.html.erb

```diff
@@ -1,7 +1,11 @@
 <div class="citations">
     <% if Hyrax.config.citations? %>
+      <h3>APA</h3>
+      <%= export_as_apa_citation(@presenter) %>
       <h3><%= t('.header') %></h3>
```

Comment:
Lets convert all this to the same method. Either all in partial or all in translations. Translations is probably better.